arena.js

```diff
@@ -0,0 +1,46 @@
+class Arena{
+  constructor(name){
+    this.name = name;
+    this.gladiator= [];
+  }
+
+  addGladiator(x){
+    if (this.gladiator.length < 2){
+      this.gladiator.push(x);
+      console.log("ok");
+    }
+  }
+
+  fight(){
+    if (this.gladiator.length == 2){
+      var gladOne = this.gladiator[0].weapon;
+      var gladTwo = this.gladiator[1].weapon;
+      if ( (gladOne =="Trident" && gladTwo == "Spear") ||
+      (gladOne == "Spear" && gladTwo == "Club") ||
+      (gladOne == "Club" && gladTwo =="Trident")) {
+        alert(`${this.gladiator[0].name} wins!`);
```

Comment:
this is great, but can you think of any way to abstract the winning conditions here so you do not have as many conditionals?